packages/shared/package.json

```diff
@@ -117,6 +117,7 @@
     "lodash.clonedeep": "^4.5.0",
     "node-fetch": "^2.6.6",
     "react-onesignal": "^2.0.4",
+    "tippy.js": "^6.3.1",
```

Comment:
Added as dependency just so we can import inside `ChangelogTooltip` to access `sticky` plugin. It does not add anything to bundle size because `tippy.js` was already a dependency of `@tippyjs/react`, it was just a matter of linter complaining.